process_manager/templates/process_manager/index.html

```diff
@@ -171,13 +206,28 @@ <h5>Process Control</h5>
                         class="btn btn-info w-100 ms-2"
                         _="on click hide me toggle .hide-messages on #main-content">Show Messages</button>
               </div>
-              <input class="form-control mb-3"
-                     type="search"
-                     name="search"
-                     placeholder="Search processes..."
-                     hx-get="{% url 'process_manager:process_table' %}"
-                     hx-trigger="input changed delay:500ms, every 1s"
-                     hx-target="div.table-container">
+              <div class="d-flex align-items-center mb-3">
+                <select id="search-dropdown" name="search-drp" class="form-control me-2">
+                  <option value="">Search By</option>
+                  <option value="uuid">UUID</option>
+                  <option value="name">Process Name</option>
+                  <option value="user">User</option>
+                  <option value="session">Session</option>
+                  <option value="status_code">Status</option>
+                  <option value="exit_code">Exit Code</option>
+                </select>
+                <input id="search-input"
+                       class="form-control"
+                       type="search"
+                       name="search"
+                       placeholder="Search processes..."
+                       style="flex: 1"
+                       hx-get="{% url 'process_manager:process_table' %}"
+                       hx-trigger="input changed delay:500ms, every 1s"
+                       hx-target="div.table-container"
+                       hx-include="#sort-field, #combined-search, #search-input, #search-dropdown">
```

Comment:
Combined search does not exist anymore, and if it does, it is not needed.
```suggestion
                       hx-include="#sort-field, #search-input, #search-dropdown">
```